catalog/schematron/catalog-schematron-plugin/src/main/java/ddf/services/schematron/SchematronValidationService.java

```diff
@@ -354,6 +364,39 @@ public String getOrganization() {
         return null;
     }
 
+    @Override
+    public Optional<MetacardValidationReport> validateMetacard(Metacard metacard) {
+
+        MetacardValidationReportImpl report = null;
+        try {
+            validate(metacard);
+        } catch (SchematronValidationException e) {
+            report = new MetacardValidationReportImpl();
+            Set<String> attributes = ImmutableSet.of("metadata");
+            if (e.getWarnings() != null) {
+                for (String each : e.getWarnings()) {
+                    report.addMetacardViolation(new ValidationViolationImpl(attributes,
+                            each,
+                            ValidationViolation.Severity.WARNING));
+                }
+            }
+            if (e.getErrors() != null) {
+                for (String each : e.getErrors()) {
+                    report.addMetacardViolation(new ValidationViolationImpl(ImmutableSet.of(metacard.getMetadata()),
+                            each,
+                            ValidationViolation.Severity.ERROR));
+                }
+            }
+        } catch (ValidationException e) {
+            LOGGER.warn("Exception validating metacard ID {}\n{}",
```

Comment:
Would prefer not to swallow the stacktrace here, maybe log it out at debug if it's not being logged elsewhere?